contents/page-sections/error-pages/404-page/index.tsx

```diff
@@ -2,12 +2,12 @@ import { Button, Heading, Text, VStack } from "@yamada-ui/react"
 
 const NotFound = () => {
   return (
-    <VStack alignItems="center" p="lg">
-      <Text fontSize="5xl" fontWeight="bold">
+    <VStack alignItems="center" p="lg" maxW="2xl">
```

Comment:
I don't see the need to specify maxW, as with the 500 pages.

Reply:
Fixed.